bundle/android/strings.xml

```diff
@@ -158,9 +159,7 @@
     <string name="gutenberg_native_embed_options" tools:ignore="UnusedResources">Embed options</string>
     <string name="gutenberg_native_error" tools:ignore="UnusedResources">Error</string>
     <string name="gutenberg_native_excerpt_length_words" tools:ignore="UnusedResources">Excerpt length (words)</string>
-    <string name="gutenberg_native_failed_to_insert_audio_file" tools:ignore="UnusedResources">Failed to insert audio file.</string>
     <string name="gutenberg_native_failed_to_insert_audio_file_please_tap_for_options" tools:ignore="UnusedResources">Failed to insert audio file. Please tap for options.</string>
-    <string name="gutenberg_native_failed_to_insert_media" tools:ignore="UnusedResources">Failed to insert media.</string>
```

Comment:
These strings were removed in https://github.com/WordPress/gutenberg/pull/39694, so the change is valid 👍 .